stl/inc/experimental/resumable

```diff
@@ -75,7 +75,7 @@ namespace experimental {
         // Returns a handle to a coroutine that has no observable effects when resumed or destroyed.
         return noop_coroutine_handle{};
     }
-#endif // !defined(__EDG__) && _MSC_VER >= 1928
+#endif // __EDG__

```

Comment:
I thought the trend went with `!__EDG__` as this is a rather long ifdef

Reply:
The oldest style (which I sometimes use out of habit) is to pair both `#ifdef MEOW` and `#ifndef MEOW` with `#endif // MEOW`. It's admittedly less clear than our newer arrow comments. (As usual, this is tracked by #351.)

I actually dislike pairing `#ifndef MEOW` with `#endif // !MEOW` because I believe that it creates confusion between the definedness and value domains (that is, `defined(MEOW)` is different from `MEOW`, we try to avoid "undefined expands to `0`" because that triggers a compiler warning).

But since you noticed it, I can update it to a nice arrow comment - I have no argument against doing so other than sheer inertia :joy_cat: